src/index.html

```diff
@@ -11,6 +11,29 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="search-box search-box--big" data-qa="big">
+      <label>
+        <form action="#" class="search-box__form search-box__form--big">
+          <a href="#" class="search-box__link search-box__link--big">
+          </a>
+          <input
+          data-qa="keypress"
+          class="search-box__input search-box__input--big"
+          type="search"
+          placeholder="Try &#34Los Angeles&#34">
```

Comment:
```suggestion
          <input
            data-qa="keypress"
            class="search-box__input search-box__input--big"
            type="search"
            placeholder="Try &#34Los Angeles&#34"
          >
```
Consider fixing indents according to the code style guide